### DISCUSSION TITLE
ip监听能否支持：监听全部

### DISCUSSION THREAD

**Discussion by ddns6** [messense/aliyundrive-webdav#544] [Q&A] [answered]
当前只能监听单个ipv4或ipv6，我使用的是ipv6，
每次设备ip变动都需要手动去更新。
期待支持自动监听全部地址(如GoWebDav)
谢谢！

**Comment by messense** [comment, marked as answer]
[提问的智慧](https://github.com/tvvocold/How-To-Ask-Questions-The-Smart-Way)

https://github.com/messense/aliyundrive-webdav/discussions?discussions_q=ipv6
